### DISCUSSION TITLE
Webp image render issue

### DISCUSSION THREAD

**Discussion by Alex-ScoopTix** [phantom/docs#260] [Help & Support] [answered]
### Summary

I've created some NFTs with webp images but Phantom seems to have some sort of issue rendering the images correctly, creating some artifacts where the transparency starts. It's not a problem with the image, it's fine locally and also rendered correctly on arweave, solana explorer and in solflare. It's a problem in the browser extension as well as the mobile app (although on mobile the artifacts are much smaller)

Here's a screenshot viewing the NFT through the extension
![image](https://github.com/user-attachments/assets/46c2fb1b-91e1-43e5-900b-e955ea5596d9)

Vs on explorer
https://explorer.solana.com/address/6z4uAnfjY5CptdRZgEeFwirWpuHtC5o6RJ8h6mFiGYmW?cluster=devnet

Is this a known issue and can we expect a fix or is there a workaround?

### Steps to Reproduce

Any webp image with transparent background

### Phantom Version

24.11.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @Alex-ScoopTix,

Thanks for writing in! We will investigate.

**Reply by Alex-ScoopTix** [reply]
Actually I realized this is not webp specific, this looks the same with png. The images are compressed using [sharp](https://github.com/lovell/sharp) unompressed this is not an issue.

**Reply by adamdelphantom** [reply, marked as answer]
@Alex-ScoopTix This should now be resolved. Please let me know if you see issues going forward.
